delfin/exception.py

```diff
@@ -170,6 +170,42 @@ class VolumeNotFound(NotFound):
     msg_fmt = _("Volume {0} could not be found.")
 
 
+class StorageHostInitiatorNotFound(NotFound):
+    msg_fmt = _("Storage host initiator{0} could not be found.")
```

Comment:
`initiator{0}` should be `initiator {0}`

Reply:
Updated